presto-docs/src/main/sphinx/sql/delete.rst

```diff
@@ -30,3 +30,9 @@ Delete all line items for low priority orders::
 Delete all orders::
 
     DELETE FROM orders;
+
+
+Limitations
+-------------
+Only :doc:`../connector/hive` currently supports DELETE and it works for the queries where one or more partitions are
+deleted entirely.
```

Comment:
Is this actually true? Say I have monthly partitions but perform a delete that removes entire year (... WHERE date_year = 2014). That satisfied your description but I think the actual implementation looks at the actual partitioning column and requires that the predicate is just on that column, nothing else. Please double check.